ThunderCloud/AccordionTabBarViewController.swift

```diff
@@ -311,14 +312,26 @@ open class AccordionTabBarViewController: TableViewController, StormObjectProtoc
         navigationController.view.sendSubviewToBack(navigationController.navigationBar)
     }
     
+    private var navigationBarHeight: CGFloat {
```

Comment:
Can we not use safe areas? Constraining to `superview` vs `safeArea` where required?

Reply:
Unsure if safe areas will satisfy our requirements... This is used to provide the height of one particular view within a stack view within the cells that make up the left hand panel of the accordion view. The navigation bar is completely hidden, so wouldn't really want to rely on safe areas (And wouldn't be sure exactly where to access it from e.t.c)

Reply:
Sure :) Let's go with this then! :P